correios/client.py

```diff
@@ -105,6 +105,15 @@ def build_tracking_codes_list(self, response):
         codes = response.split(",")
         return TrackingCode.create_range(codes[0], codes[1])
 
+    def _load_invalid_event(self, tracking_code: TrackingCode, tracked_object):
+        event = TrackingEvent(
+            timestamp=datetime.now().strftime("%d/%m/%Y %H:%M"),
+            status=EventStatus("ERROR", 1),
+            location_zip_code="00000-000",
+            comment=tracked_object.erro,
+        )
+        tracking_code.add_event(event)
```

Comment:
Esse formato da data é específico ou poderia ser um .isoformat() ?
Se for específico, acho que poderia ser uma configuração tipo "DATETIME_FORMAT", provavelmente deve ter se repetido em algum outro ponto.

Outra questão sobre a data, ela é naive mesmo ? Não precisaria ser utc ?

O "location_zip_code" não poderia ser None ou "" ?